test/unit/alignment/pairwise/policy/affine_gap_policy_test.cpp

```diff
@@ -19,8 +19,8 @@
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/std/ranges>
 
-using namespace seqan3;
-using namespace seqan3::detail;
+using seqan3::operator""_dna4;
+using seqan3::operator|;
```

Comment:
why do you need this?

Reply:
Because of:
```bash
.../seqan3/test/unit/alignment/pairwise/policy/affine_gap_policy_test.cpp:137:62: error: no match for 'operator|' (operand types are 'seqan3::detail::trace_directions' and 'seqan3::detail::trace_directions')
  137 |                   seqan3::detail::trace_directions::diagonal | seqan3::detail::trace_directions::up_open
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                     |                                            |
      |                                                     seqan3::detail::trace_directions             seqan3::detail::trace_directions

```

Reply:
@rrahn Is this expected? It seems like a bug

Reply:
This is the `add_enum_bitwise_operators` bug that we discussed in https://github.com/seqan/seqan3/wiki/Core-Meeting-notes#2020-02-17-eseiler

The problem is that all operators like `operator|` that are enabled by `add_enum_bitwise_operators`  are in seqan3 namespace. `trace_directions` is in namespace `seqan3::detail` and thus will not find those operators by ADL, because they are in a different namespace.

Reply:
ah ok thanks :)